File: fizzBuzzes/fizzBuzz.js
// JavaScript

function fizzBuzz(n) {
  for (var i = 1; i <= n; i += 1) {
    if (i % 5 == 0 && i % 3 == 0) {
      console.log('fizzBuzz');
    } else if (i % 5 == 0) {
      console.log('fizz');
    } else if (i % 3 == 0) {
      console.log('buzz');
    } else {
      console.log(i);
    }
  }
}

fizzBuzz(16)

File: fizzBuzzes/fizz_buzz.rb
# Ruby

def fizz_buzz n
  for i in 1..n
    if i % 3 == 0 && i % 5 == 0
      puts 'fizz_buzz'
    elsif i % 3 == 0
      puts 'fizz'
    elsif i % 5 == 0
      puts 'buzz'
    else
      puts i
    end
  end
end

fizz_buzz 16
